scripts/upload_zoom_recordings.py

```diff
@@ -61,8 +62,10 @@ def download_file(url, download_path):
 
         print('Recording is permitted for upload!')
         for file in meeting['recording_files']:
+            uploaded_files = 0
             if file['file_size'] == 0:
                 print('Meeting still processing: {}'.format(meeting['topic']))
+                requests.post(NOTIFICATION_WEBHOOK_URL, {'status':'processing'})
```

Comment:
This raises an exception if you pass `False` (the default) for the URL. Should it be guarded by an `if` statement?